services/redis.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sd-redis
+  labels:
+    name: redis
+    app: sd-redis
+spec:
+  ports:
+  - port: 6379
+    name: master
+  clusterIP: None
```

Comment:
We don't need load balancing with this Service since there will always be one master https://kubernetes.io/docs/concepts/services-networking/service/#headless-services